jbmc/unit/java-testing-utils/require_type.h

```diff
@@ -28,6 +28,9 @@ require_pointer(const typet &type, const optionalt<typet> &subtype);
 const struct_tag_typet &
 require_struct_tag(const typet &type, const irep_idt &identifier = "");
 
+const pointer_typet
+require_pointer_to_tag(const typet &type, const irep_idt &identifier = "");
```

Comment:
Please use `irep_idt()` instead of `""`. (#4041 cleans this up for existing code.)

Reply:
Done